pandas/core/arrays/base.py

```diff
@@ -1646,10 +1646,12 @@ def __array_ufunc__(self, ufunc: np.ufunc, method: str, *inputs, **kwargs):
 
 
 class ExtensionArraySupportsAnyAll(ExtensionArray):
-    def any(self, *, skipna: bool = True) -> bool:
+    def any(self, *, skipna: bool = True) -> bool:  # type: ignore[empty-body]
```

Comment:
Another option I think is to make these abstract methods - I haven't checked if that's viable, and maybe not something you'd want to handle here.

Reply:
Would be ok with doing that, but would like to keep things separated